lib/thrift/generator/utils.ex

```diff
@@ -60,30 +60,47 @@ defmodule Thrift.Generator.Utils do
   end
 
   @doc """
-  Merge the binaries in an iolist.
+  Optimize an expression that returns an iolist.
 
-    ["a", "b", ["c", [var]]] => ["abc", var]
+    [<<1>>, <<2>>] => <<1, 2>>
```

Comment:
That makes sense. You can use standard comments to describe what's going on here for internal consumption.